erpnext/hooks.py

```diff
@@ -262,6 +262,7 @@
 		"validate": "erpnext.regional.india.utils.validate_tax_category"
 	},
 	"Sales Invoice": {
+		"after_insert": "erpnext.regional.create_qr_code",
```

Comment:
`erpnext.regional.create_qr_code` can be moved to `erpnext.regional.saudi_arabia.utils.create_qr_code`?

As we have namespaced all the Indian helper functions to `regional.india`.

Reply:
I saw similar code in __init__.py for `["Nepal", "France"]`  and for `["France", "Germany"]`
Anyways moved it as suggested